src/diffusers/pipelines/pipeline_loading_utils.py

```diff
@@ -369,6 +488,7 @@ def load_sub_model(
     provider: Any,
     sess_options: Any,
     device_map: Optional[Union[Dict[str, torch.device], str]],
+    device_map_strategy: Optional[str],
```

Comment:
I think we can remove `device_map_strategy`.  We don't need it after the allocation is done (`device_map` as a dict). 
```suggestion
```

Reply:
Done in fc9b2209.